bower.json

```diff
@@ -19,22 +19,11 @@
   ],
   "main": "dist/angularfire.js",
   "ignore": [
-    "**/.*",
-    "src",
-    "tests",
-    "node_modules",
-    "bower_components",
-    "firebase.json",
-    "package.json",
-    "Gruntfile.js",
-    "changelog.txt"
+    "**/*",
+    "!dist/angularfire.js"
```

Comment:
Didn't know you could do this! Interesting. That being said, I don't think this is entirely correct. It looks like this won't contain things like the README, the license, the minified dist file, and the `index.js` file.